.github/workflows/community-releases.yml

```diff
@@ -42,8 +42,7 @@ jobs:
 
       - name: Install pnpm
         uses: pnpm/action-setup@v3
-        with:
-          version: 8.9.0
+
```

Comment:
We don't need these - PNPM 9 now uses the `packageManager` entry of the root package.json to determine the version to use. It downloads this version on the fly via corepack configuration of `pnpm/action-setup`